client/keys/codec.go

```diff
@@ -5,6 +5,8 @@ import (
 	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
 )
 
+// TODO: remove this file #8047
```

Comment:
nit: post whole issue url to easily access

Reply:
ok